docs/querying/sql.md

```diff
@@ -709,6 +709,8 @@ Druid does not support all SQL features. In particular, the following features a
 
 - JOIN between native datasources (table, lookup, subquery) and system tables.
 - JOIN conditions that are not an equality between expressions from the left- and right-hand sides.
+- JOIN containing constant value inside the condition
```

Comment:
I think this limitation is implied from the above limitation. Slight grammar re-wording if you want to keep the warning.
```suggestion
- JOIN conditions containing a constant value inside the condition.
```

Reply:
Just to be explicit since the mentioning of "expressions" on the above limitation may be missed.

Reply:
Done